hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapMergeContainer.java

```diff
@@ -109,6 +109,6 @@ public void readData(ObjectDataInput in) throws IOException {
         creationTime = in.readLong();
         lastAccessTime = in.readLong();
         lastUpdateTime = in.readLong();
-        hits = in.readInt();
+        hits = in.readLong();
```

Comment:
i believe no backward compatibility problem exists since merging is new for multimap?

Reply:
Yes, I've added the `@since 3.10` to the class, to make it clearer.